src/styles/blocks/search-bar--big.css

```diff
@@ -0,0 +1,15 @@
+.search-bar--big {
+  height: 70px;
```

Comment:
Avoid fixing the height of elements if there is no specific requirement for it. This can lead to content overflow or unexpected layout issues when the content changes or scales. Let the content dictate the size of its container.